src/components/NonIdealState.jsx

```diff
@@ -0,0 +1,11 @@
+import React from "react";
+
+const NonIdealState = () => 
+<div className="vizbuilder-nonidealstate">
+  <h1 className="vizbuilder-nonidealstate-header">
+    {"No results"}
```

Comment:
This label should use the translation hook to get localization.
File /src/components/IssueButton.jsx is a brief example of how to use it.
Base labels file is /src/toolbox/useTranslation.js
Also, line 8 seems to be an extra `}`.